rskj-core/src/main/java/co/rsk/scoring/InetAddressUtils.java

```diff
@@ -11,6 +11,8 @@
  * Created by ajlopez on 15/07/2017.
  */
 public class InetAddressUtils {

```

Comment:
Maybe declare the class final? https://stackoverflow.com/questions/7486012/static-classes-in-java